Software Requirements Specifications.txt

```diff
@@ -5,8 +5,14 @@ Goals and Requirements
 •	1.0 There shall be at minimum two tanks.
 •	1.0.1 The tanks shall be able to traverse a vertical height of one block.
 •	1.0.2 The tanks shall be able to survive at least 5 collisions with the projectile block.
+•	1.0.3 One tank shall be the user and the other tank shall be a NPC.
 •	1.1 The game shall have destructible terrain.
 •	1.2 The tanks shall shoot projectiles to attack.
 •	1.2.1 A new projectile block will be implemented.
 •	1.2.2 The projectiles shall destroy surrounding terrain.
 •	1.2.3 The projectile will have at minimum a range of 15 blocks.
+•	1.3 When the user types,"/play tank minigame" into the command window the tank game shall begin
+•	1.3.1 The user will be spawned into an arena as a tank.
+•	1.3.2 A enemy tank NPC will spawn in the arena after 10 seconds.
+•	1.3.3 The arena will be on a preset island.
+•	1.3.4 The first to score 5 hits on the other tank will win the minigame and the user will be taken back to the last location they were at before playing the tank minigame.
```

Comment:
I think we should split 1.3.4 into 2 different requirement, because of the implicit ands.
1.3.4 The first to score 5 hits on the other tank will win the minigame
1.3.5 After the minigame the user will be taken back to the last location they were at before playing the tank minigame

Reply:
I agree with Jonathan to change 1.3.4 into 2 different requirements, because it can be broken down into smaller requirements making them easier to meet.